src/frequently-requested-changes.md

```diff
@@ -214,6 +214,29 @@ of tail or internal padding, as they can be reused due to the lack of a possible
 
 Cross-referencing to other discussions:
 
-* https://github.com/rust-lang/rfcs/issues/1397
-* https://github.com/rust-lang/rust/issues/17027
-* https://github.com/rust-lang/unsafe-code-guidelines/issues/176
+* <https://github.com/rust-lang/rfcs/issues/1397>
+* <https://github.com/rust-lang/rust/issues/17027>
+* <https://github.com/rust-lang/unsafe-code-guidelines/issues/176>
+
+## A way to bypass visibility, including a `unsafe` bypass
+
+Items are only accessible if they are marked `pub` or re-exported as such,
+they are otherwise private by default. People sometimes wish to break that
+rule to access internals of libraries they're using, for example to access
+private fields of a type or to call private functions.
+
+This could break invariants assumed by the crate's author, which if any
+unsafe code depends on those could lead to undefined behavior.
```

Comment:
```suggestion
This could break invariants assumed by the crate's author, which, if any
unsafe code depends on those, could lead to undefined behavior.
```